### PR TITLE
Add component buttons

### PR DESCRIPTION
**Pull request by sah-shah7**
-Added Primary & Link Buttons
-Added Buttons with Icon
-Added Floating Buttons
image
image
image

### CODE REVIEW

File: Components/Alerts/alerts.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                    style="width: 1024px; height: 1082px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (som749898) [inline Components/Alerts/alerts.html:158]: Don't use inline styling

**Review by som749898 — COMMENTED** [review]
use CSS variable for color
Avoid using inline styling
Try to have better class name

File: Components/Alerts/alert.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
+
+.alert {
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 5px;
+  padding: 1.25rem 2.5rem;
+  box-shadow: 0 0 15px 5px #ccc;
+  border-left: 0.5rem solid;
```
Comment (AtulHande03a) [inline Components/Alerts/alert.css:8]: using rem & px for small values.. stick to one .. and for less than 20px you can use px.

File: Components/Alerts/alert.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  cursor: pointer;
+}
+
+.close-btn .fa-times {
+  font-size: 1.375rem;
+  line-height: 2.5rem;
+}
+
+.alert-warning {
+  background-color: #fff3cd;
```
Comment (AtulHande03a) [inline Components/Alerts/alert.css:36]: variable names for colors

File: Components/Alerts/alerts.html [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+                        <a href="../../Components/modal/modal.html" class="link-a link-a-font-color-black sidebar-list-items-text">Modal</a>
+                    </li>
+                    <li class="sidebar-list-items">
+                        <a href="../../Components/navigation/navigation.html" class="link-a link-a-font-color-black sidebar-list-items-text">Navigation</a>
+                    </li>
+                    <li class="sidebar-list-items">
+                        <a href="../../Components/rating/rating.html" class="link-a link-a-font-color-black sidebar-list-items-text">Rating</a>
+                    </li>
+                    <li class="sidebar-list-items">
+                        <a href="../../Components/grid/grid.html" class="link-a link-a-font-color-black sidebar-list-items-text">Simplified Grid</a>
```
Comment (AtulHande03a) [inline Components/Alerts/alerts.html:84]: not the correct convention to name classes .. top big names for classes

**Review by AtulHande03a — CHANGES_REQUESTED** [review]
apart from comments rest looks ok!!

File: Components/Alerts/alerts.html (hunk 1 above)
Comment (sah-shah7) [reply Components/Alerts/alerts.html:158]: Hey, This iframe style is injected by carbon highlight. I have not added inline style.

File: Components/Alerts/alert.css (hunk 2 above)
Comment (sah-shah7) [reply Components/Alerts/alert.css:8]: Will make the changes, thanks

**Comment by sah-shah7** [comment]
CSS color variable have been added in a branch ahead of this. PR approved. Merging

**Pull request merged**